Mass: 100 # kg
Radius: 0.100 # m
GAccel: 10 # m/s^2
kForce: 1 # kg*m
kTorque: 1 # kg*m^2
